package.json

```diff
@@ -184,6 +184,20 @@
       "type": "string",
       "value": "[0.4, 0.7, 0.1, -0.4, -0.2, -0.1]",
       "hidden": true
+    },
+    {
+      "name": "metadataRatingSystem",
+      "title": "Rate this metadata",
+      "type": "bool",
+      "value": false,
+      "hidden": true
+    },
+    {
+      "name": "metadataSource",
```

Comment:
this pref will come in handy for future patches to dynamically set the metadata endpoint based on the source, and also to store it nicely in the metadata db as an easily identifiable source